stable_baselines/a2c/a2c.py

```diff
@@ -335,6 +347,10 @@ def run(self):
                 maybe_ep_info = info.get('episode')
                 if maybe_ep_info is not None:
                     ep_infos.append(maybe_ep_info)
+                if info.get('valid_actions') is not None:
+                    action_mask = np.expand_dims(np.array(info.get('valid_actions'), dtype=np.bool),
```

Comment:
Seems like this line and the current version of reshape_action_masks don't work together.  reshape_action_masks seems to filter by numerical value to determine which indexes to set to -infinity, and is causing the action_masks to not get set.

Reply:
Changing to np.float returns what I think looks like expected values for reshape_action_masks, but then something later on is causing all actions to be 0.

Reply:
You're right. I forgot to update this np.array type to be float after my multiplication -> addition change.